test/queries/developer_query_test.rb

```diff
@@ -126,7 +132,8 @@ class DeveloperQueryTest < ActiveSupport::TestCase
       utc_offsets:,
       role_types: [:part_time_contract],
       role_levels: [:junior],
-      include_not_interested: true
+      include_not_interested: true,
+      search_query: ""
```

Comment:
Let's add some query here so it tests the value is getting passed along correctly.